### ISSUE TITLE
Wrong indent when `>` is in code

### ISSUE THREAD

**Issue by MMauro94** [JetBrains/kotlin-playground#59]
I think I've found a bug: when using the `from` and `to` parameters with the character `>` is present in the code the indents are screwed up.

See attached HTML for example.

[test.zip](https://github.com/JetBrains/kotlin-playground/files/3331027/test.zip)

**Comment by AlexanderPrendota** [comment]
Hey! Thank you for the report. Yes, you are right. 
it's due to `//Doing things` after `>`.
As I understand there is a bug in CodeMirror library in [`clike`](https://github.com/codemirror/CodeMirror/blob/master/mode/clike/clike.js#L637) mode. :(

**Comment by MMauro94** [comment]
> it's due to `//Doing things` after `>`

What do you mean? When removing the comment (and leaving just the two prints) the wrong indent is still present on the first line of the lambda.

  
> As I understand there is a bug in CodeMirror library in clike mode. :(

Should I report this bug to them then?

**Comment by AlexanderPrendota** [comment]
[test.html.zip](https://github.com/JetBrains/kotlin-playground/files/3392417/test.html.zip)
It works without a comment line on my side

**Comment by MMauro94** [comment]
You forgot to change the `to` value to `5`, since we've removed a line.

[test.zip](https://github.com/JetBrains/kotlin-playground/files/3392740/test.zip)

**Comment by AlexanderPrendota** [comment]
Ah! I see! Thanks! Yep, I reproduced it. 
So, really, there is a problem with auto-indent in codemirror.
Try to set the attribute `auto-indent="true"` and remove attributes `to` & `from`.

**Comment by MMauro94** [comment]
It works by removing `to` and `from` and setting `auto-indent` to **`false`**. Any other combination doesn't work. The problem is that I need the `from` and `to` parameters to hide some reduntant code and imports, so I'll have to deal with the wrong indent for the time being I guess.

Also, I discovered that the issue appears only when either character `-` or `>` are in the code, but not if it's in a comment. (see test)

[test.zip](https://github.com/JetBrains/kotlin-playground/files/3401177/test.zip)

**Comment by AlexanderPrendota** [comment]
false value - default. Try:
```html
<code class=".code" auto-indent="true">
fun main() {
    listOf(1L, 2L).forEach { it ->
        print("Hello ")
	println(it)
    }
}
</code>
```

**Comment by AlexanderPrendota** [comment]
> Any other combination doesn't work.
Yes. There is an issue in clike mode in Codemirror =(

**Comment by AlexanderPrendota** [comment]
> I discovered that the issue appears only when either character - or > are in the code,

Yes. U can see how auto-indent works [here](https://github.com/codemirror/CodeMirror/blob/17457466015551c7bf9292896baaa69e738f0d49/mode/clike/clike.js#L679)

**Comment by MMauro94** [comment]
Your provided example still has the issue.

> Yes. There is an issue in clike mode in Codemirror =(

Should I open an issue in Codemirror then?

> Yes. U can see how auto-indent works here

Apparently it detects the last character (`>`), thinks it's an operator and for this reason adds the indent.
